### DISCUSSION TITLE
单位中的希腊字母显示不出来

### DISCUSSION THREAD

**Discussion by Tsiword** [tuna/thuthesis#697] [Q&A] [answered]
使用了siunitx包，但是latex源码输入$\SI{4.726}{\mu W}$或者\SI{4.726}{\mu W}时，都显示4.726￿W。
希腊字母变成了方框。
在普通公式环境下希腊字母可以正常显示。

**Comment by zepinglee** [comment, marked as answer]
阅读 `siunitx` 的文档，正确的用法是 `\micro`。
